### DISCUSSION TITLE
YAML dates for multiple entries.

### DISCUSSION THREAD

**Discussion by Schachte** [juftin/camply#269] [Q&A] [answered]
First off, great app. This documentation is awesome.

I want to get 2 dates that are either:
- Night 1
- Night 2
- Night 1 and Night 2

I'm using YAML config:

```
start_date: 
  - 2023-06-23 
  - 2023-06-24 
end_date: 
  - 2023-06-24 
  - 2023-06-25 
nights: 1
```

How should I best go about this?

**Comment by juftin** [comment, marked as answer]
🤔 @Schachte it seems like a `1` night reservation or a `2` night reservation would work for you as long as it's between `2023-06-23` and `2023-06-25`. If that's right, I would remove `nights` from the YAML body entirely (I'm leaving it in below because `1` is the default anyways). Camply won't combine reservations into two night bundles, instead it would send you two notifications (one for each night). 

```yaml
start_date: 2023-06-23 
end_date: 2023-06-25
nights: 1
```
